external_libraries/CMakeLists.txt

```
@@ -6,6 +6,11 @@ if (NOT EXISTS ${CMAKE_CURRENT_SOURCE_DIR}/nova-simd/vec.hpp)
 			" https://github.com/supercollider/supercollider/wiki/git-cheat-sheet#working-with-submodules")
 endif()
 
+if(NOT SYSTEM_PORTAUDIO)
+    message(STATUS "PortAudio: building static library")
+    add_subdirectory("${CMAKE_SOURCE_DIR}/external_libraries/portaudio")
```

sorry, just noticed this--

```suggestion
    add_subdirectory(portaudio)
```